app/views/admin/contest_groups/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1>Групи</h1>
+<br />
+
+<% if !@contest_groups.blank? %>
```

Comment:
Generally using `!` in the conditionals may lead to confusion sometimes. Rails provides things like `.present?` to avoid using `!`. It is not that important, so change it only if you want to :)